src/engraving/dom/textbase.cpp

```diff
@@ -1892,7 +1892,8 @@ void TextBase::createBlocks(LayoutData* ldata) const
 
                         //char32_t code = score()->scoreFont()->symCode(id);
                         char32_t code = id
-                                        == SymId::space ? static_cast<char32_t>(' ') : score()->engravingFonts()->fallbackFont()->symCode(id);
+                                        == SymId::space ? static_cast<char32_t>(' ')
+                                        : score()->engravingFonts()->fallbackFont()->symCode(id);
```

Comment:
Another unrelated change; I won't comment on all of them, but it'd be good to do a quick pass over all files to ensure that there are no such changes